src/akkudoktoreos/class_pv_forecast.py

```diff
@@ -259,7 +259,7 @@ def __init__(
         else:
             self._forecast_start = None
         self.prediction_hours = prediction_hours
-        self._tz_name = None
+        self._tz_name = "Europe/Berlin"
```

Comment:
instead of hard coding the time zone to Germany, would it make better sense to determine the OS configured time zone and use that?
Or at least move the time zone as a parameter to the JSON config?

Reply:
json is obviously the way to go. i had made a future todo list in an edit in the first comment with this. looks like i didn't hit save. I have not yet worked with the new cfg. if its ok for you i would make a later PR and change much more stuff into the cfg and not only this.

Reply:
to_timezone in dateimeutils.py will provide you a timezone based on latitude and longitude. We need latitude and longitude anyway for PV forecast. Currently it is hidden in the url provided for forecast request.
I'm working on to make the forecasts more universal.